packages/react-client/src/ReactFlightClient.js

```diff
@@ -1227,6 +1247,20 @@ function parseModelString(
         }
         return readTemporaryReference(temporaryReferences, reference);
       }
+      case '*': {
+        // Lazy Halted Reference
+        if (enableHalt) {
+          return createLazyChunkWrapper(createBlockedChunk<mixed>(response));
+        }
+        // fallthrough
+      }
+      case '&': {
+        // Eager Halted Reference
```

Comment:
Nit but at this level we’re not talking about Halting anymore. It’s just an infinitely blocking reference. Halting is just one mechanism that generates these references. Just like we already had infinite promises so it’s not a halted promises it’s a more general feature we might need for other things.